strings/src/main/java/com/revenuecat/purchases/strings/BillingStrings.kt

```diff
@@ -15,6 +15,7 @@ object BillingStrings {
         "PurchaseHistoryRecord, but only one will be used."
     const val BILLING_PURCHASE_MORE_THAN_ONE_SKU = "There's more than one sku in the PurchaseHistoryRecord, " +
         "but only one will be used."
+    const val CANNOT_CALL_METHOD_FROM_STORE = "%s cannot be called from %s. %s"
```

Comment:
"canMakePayments cannot be called from AMAZON_STORE. Returning false" is not super clear to me. I think the "from" confuses me. Do you have an alternative message?